Sources/MongoSwift/Operations/StartSessionOperation.swift

```diff
@@ -48,12 +48,10 @@ internal struct StartSessionOperation: Operation {
         self.session = session
     }
 
-    internal func execute(using connection: Connection, session _: ClientSession?) throws {
-        // session was already started
-        guard case let .notStarted(opTime, clusterTime) = self.session.state else {
```

Comment:
we already check the state before we fire off this operation. assuming users aren't concurrently using the session across threads (not allowed) there's no way for it to change between then and when execution happens here.